README.md

```diff
@@ -21,25 +21,38 @@ Can be installed via `pip`: `pip install codeflare-sdk`
 ### Prerequisites
 
 We recommend using Python 3.9 for development.
-Install development specific dependencies:
-  `$ pip install -r requirements-dev.txt`
 
-Additional dependencies can be found in `requirements.txt`: `$ pip install -r requirements.txt`
+#### Install dependencies:
+  - run: `poetry install`
```

Comment:
I wonder if we should instead use `poetry shell` and go from there. Might be less annoying than repeatedly adding `poetry run`

Reply:
Updated. Thanks Anish.